ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+
+This is a brief version of the report that represents information obtained from experts, aggregated by the end of June 2024. Part of the information and recommendations received from experts have not yet been aggregated and are not included in this report. The research took over 2 months almost full-time, while Sergey was only paid for one month. The rest of the information will be aggregated and provided later if Sergey has time or if he is paid for it.
+
+## Investments
+Ubiquity's ability to retain the amount of investments received in the previous round commands deep respect. Such a situation is a strong positive signal to investors that their funds are managed efficiently. It is desirable for such information to be confirmed by facts that investors can verify.
+
+### Funding Round
+Since Ubiquity has funds from the previous round, it remains unclear for what purposes to attract a new investment round. To justify the need for additional funding, it is necessary to provide a reasoned development strategy for Ubiquity with directions for using capital.
+
+Typically, more effective times for investments are autumn and spring, summer being the least active period. It is better to start contacting investors at the beginning of autumn 2024.
+
+### Investors
+Experts asked whether Ubiquity is approaching current investors with an offer to participate in a new funding round. If investors familiar with Ubiquity are expanding their investments, this is a positive signal. If none of the existing investors wish to expand their investments in Ubiquity, this could be a negative signal. Investors communicate with each other and will try to get feedback about Ubiquity from current investors.
+
+### Valuation
+One of the approaches to determining valuation is market references. Experts listed several companies with which they will compare Ubiquity to determine valuation:
+
+- Tea protocol
+
+[https://tea.xyz/](https://tea.xyz/)
+
+- Talent protocol
+
+[https://www.talentprotocol.com/](https://www.talentprotocol.com/)
+
+[https://www.build.top/](https://www.build.top/)
+
+- BuildAI
+
+[https://buildaierc.com/](https://buildaierc.com/)
+
+## Tokenomics
+Tokenomics is one of the most important parts of a project evaluation. From the experts' point of view, the tokenomics of Ubiquity's tokens are opaque and risky. Although the distribution of UBQ tokens is unclear, presumably high inflation of UBQ tokens could lead to a decrease in the value of investments. Investors need to know the main parameters - token distribution, conditions (cliff, vesting, etc), and expected valuation. 
```

Comment:
I generally explain that it was officially, originally a "fair launch" and that 100% of the supply comes from LP incentives. However, 33% is issued to the protocol admin wallet which we can use for strategic objectives, such as fundraising.